<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="5" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="170065" started-at="2019-11-13T12:47:08 IST" finished-at="2019-11-13T12:49:58 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="170065" started-at="2019-11-13T12:47:08 IST" finished-at="2019-11-13T12:49:58 IST">
      <class name="com.atmecs.testscript.TutorialsNinja">
        <test-method status="PASS" signature="startBrowser()[pri:0, instance:com.atmecs.testscript.TutorialsNinja@6c6cb480]" name="startBrowser" is-config="true" duration-ms="14527" started-at="2019-11-13T12:47:08 IST" finished-at="2019-11-13T12:47:23 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startBrowser -->
        <test-method status="FAIL" signature="validateHomepage()[pri:0, instance:com.atmecs.testscript.TutorialsNinja@6c6cb480]" name="validateHomepage" duration-ms="136" started-at="2019-11-13T12:47:23 IST" finished-at="2019-11-13T12:47:23 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [The Ninja Stor] but found [The Ninja Store]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [The Ninja Stor] but found [The Ninja Store]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.atmecs.testscript.TutorialsNinja.validateHomepage(TutorialsNinja.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateHomepage -->
        <test-method status="FAIL" signature="addToCart()[pri:1, instance:com.atmecs.testscript.TutorialsNinja@6c6cb480]" name="addToCart" duration-ms="26" started-at="2019-11-13T12:47:23 IST" finished-at="2019-11-13T12:47:23 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.atmecs.helper.Helper.clickElement(Helper.java:61)
at com.atmecs.testscript.TutorialsNinja.addToCart(TutorialsNinja.java:47)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addToCart -->
        <test-method status="PASS" signature="verifyProductInCart()[pri:2, instance:com.atmecs.testscript.TutorialsNinja@6c6cb480]" name="verifyProductInCart" duration-ms="90577" started-at="2019-11-13T12:47:23 IST" finished-at="2019-11-13T12:48:54 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyProductInCart -->
        <test-method status="FAIL" signature="validateProductDetail()[pri:3, instance:com.atmecs.testscript.TutorialsNinja@6c6cb480]" name="validateProductDetail" duration-ms="3671" started-at="2019-11-13T12:48:54 IST" finished-at="2019-11-13T12:48:57 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.atmecs.utils.ExcelReader.getData(ExcelReader.java:68)
at com.atmecs.testscript.TutorialsNinja.validateProductDetail(TutorialsNinja.java:84)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateProductDetail -->
        <test-method status="PASS" signature="verifyGrandTotal()[pri:4, instance:com.atmecs.testscript.TutorialsNinja@6c6cb480]" name="verifyGrandTotal" duration-ms="60277" started-at="2019-11-13T12:48:57 IST" finished-at="2019-11-13T12:49:58 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyGrandTotal -->
        <test-method status="PASS" signature="endBrowser()[pri:0, instance:com.atmecs.testscript.TutorialsNinja@6c6cb480]" name="endBrowser" is-config="true" duration-ms="770" started-at="2019-11-13T12:49:58 IST" finished-at="2019-11-13T12:49:58 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endBrowser -->
      </class> <!-- com.atmecs.testscript.TutorialsNinja -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
